app/src/main/java/com/example/marvelapp/data/local/model/CharacterRealmEntity.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.marvelapp.data.local.model
+
+import io.realm.RealmList
+import io.realm.RealmObject
+import io.realm.annotations.PrimaryKey
+
+open class CharacterRealmEntity(
+    @PrimaryKey
+    var id: Int = 0,
```

Comment:
Here you can create a default const val... because for example if in the future I said the default value of response is gonna be "Not Value" i need to change in a lot of places instead of you create a const.

Reply:
Thx, now its not hardcoded anymore

![image](https://user-images.githubusercontent.com/73836200/111638559-32828680-87d9-11eb-942e-7d2339fd86b5.png)